resources/js/components/ui/form/util.ts

```diff
@@ -3,11 +3,19 @@
  *
  * Doing this on the frontend is more reasonable than doing this for all fields, regardless
  * of their origin, on the backend.
+ *
+ * This function also removes double spaces from the text.
  */
 export function trimInput(inputValue: string) {
-  return inputValue.trim();
+  return inputValue.trim().replaceAll(EXTRA_SPACES_REGEXP, "$1");
 }
 
+/**
+ * Matches whitespace at line start and end, and spaces after whitespace.
```

Comment:
po co "start and end" skoro już jest trim() ?

Reply:
No bo chodzi o linie, również w wieloliniowym tekście.